src/nisarqa/processing/insar/quiver_plots.py

```diff
@@ -55,24 +62,39 @@ def plot_offsets_quiver_plot_to_pdf(az_offset, rg_offset, params, report_pdf):
         A structure containing processing parameters to generate quiver plots.
     report_pdf : matplotlib.backends.backend_pdf.PdfPages
         The output PDF file to append the quiver plot to.
+    quiver_projection_params : None or ParamsForAzRgOffsetsToProjected, optional
+        ** Strongly Recommended for GOFF and GUNW **
+        If None, or if offsets arrays are nisarqa.RadarRaster, this will
+        be ignored. Set this to None if the contents of the offset rasters
```

Comment:
Maybe we should raise an exception if the input is a `RadarRaster` and `quiver_projection_params is not None`?

(We probably don't want to raise an exception if the input array is a `GeoRaster` and `quiver_projection_params is None`, because that would preclude the possibility of generating QA plots for offsets that have already been re-projected.)